docs/articles/baseplatform/booting-qemu-image.rst

```diff
@@ -10,15 +10,35 @@ virtualization feature if found on the host system and run the virtual
 machine in better performance.
 See KVM feature for QEMU [#qemu_kvm_feature]_.
 
-Installing KVM and its dependencies
+QEMU launch script
 -----------------------------------
 
-On Debian-based systems, the following command can be used to install
-the emulator tool and its dependencies:
+There is a QEMU launch script called ``qemu_launcher``, which is shipped with SDK and also available from yocto builds.
 
 .. code-block:: bash
 
-    $ sudo apt-get install qemu-kvm libvirt-clients libvirt-daemon-system
+	$ ./qemu_launcher -h
+	usage: scripts/qemu_launcher [options]
+	[options] is any of the following:
+	  -f | --folder           	Specify a source folder of the QEMU image
+								Defaults to tmp/deploy/images/qemux86-64
+	  -i | --hda              	Specify rootfs file. Defaults to core-image-ntg-x-qemux86-64.ext4
```

Comment:
We probably should have the default be `core-image-pelux-minimal-qemux86-64.ext4` or whatever it is called, if anything.

Reply:
Yes, I updated the script in meta-pelux but forgot to update the docs. Good catch!